infrastructure/demo.tfvars

```diff
@@ -11,4 +11,5 @@ data_store_max_pool_size = 48
 database_sku_name = "GP_Gen5_8"
 database_sku_capacity = "8"
 
-definition_latest_version_ttl_sec = 30
\ No newline at end of file
+definition_latest_version_ttl_sec = 30
+definition_jurisdiction_ttl_sec = 30
```

Comment:
considering this operation is called only once when a user logs in, I doubt 30 sec ttl will have any effect on performance. It'll help only if 2 users that retrieve the same jurisdictions login max 30 sec apart from each other

Reply:
We can change this number, but the caching of the Jurisdiction list is already serving for purpose of that particular use. Approximately 3K calls per day, this optimisation can still be useful, if not sensible across the entire system.